### PR TITLE
Refresh running modules

### PR DESCRIPTION
**Pull request by asdil12**
Update the `#details` view of the test result page while the test is running when a module is finished.

### CODE REVIEW

File: assets/javascripts/running.js [hunk 1]
```diff
@@ -32,13 +32,21 @@ function updateTestStatus(newStatus) {
...
     // If a new module have been started, redraw module list
-    if (testStatus.modlist_initialized == 0 || testStatus.running != newStatus.running) {
+    if (testStatus.running != newStatus.running) {
         testStatus.running = newStatus.running;
-        $.ajax("/tests/" + testStatus.jobid + "/modlist").
-            done(function(modlist) {
-                if (modlist.length > 0) {
-                    updateModuleslist(modlist, testStatus.jobid, testStatus.running);
-                    testStatus.modlist_initialized = 1;
+        $.ajax("/tests/" + testStatus.jobid).
```
Comment (coolo) [inline assets/javascripts/running.js]: you're not seriously fetching the full HTML of tests/XXX to then throw it away?

And if /modlist is no longer needed, remove its code.

**Comment by coolo** [comment]
But the principle works. So now optimize this.

**Comment by okurz** [comment]
… I'm losing it …

**Comment by coolo** [comment]
your religion?

**Comment by okurz** [comment]
> your religion?

temper

**Comment by aaannz** [comment]
I'm with @okurz here, just not loosing the temper. :) Care to add comment? Especially since this is changing public facing routes.

**Comment by okurz** [comment]
LGTM for the better details in the commit message

**Comment by asdil12** [comment]
Test propably only fails due to new mojo version.

**Comment by okurz** [comment]
LGTM

File: assets/javascripts/running.js [hunk 2]
```diff
@@ -30,15 +31,63 @@ function updateTestStatus(newStatus) {
           return;
     }
     $('#running_module').text(newStatus.running);
+
+    // Reload broken thumbnails (that didn't exist yet when being requested)
+    if (testStatus.img_reload_time++ % 7 == 0) {
```
Comment (okurz) [inline assets/javascripts/running.js]: magic number

**Pull request merged**